tests/e2e/testfiles/minimal-testfiles/storage_csm_powerflex_auth.yaml

```diff
@@ -16,3 +16,11 @@ spec:
       # enable: Enable/Disable csm-authorization
       enabled: true
       configVersion: v2.0.0
+      components:
```

Comment:
In the components can we remove non necessary params like image ? It should be just enable TRUE or false with proxy host

Reply:
@harishp8889 the proxy host is under the component name and the env section.
This was discussed in the morning